src/app/viewer/news/news-card/news-card.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { NewsService } from 'src/app/shared/services/news.service';
+
+@Component({
+  selector: 'news-card',
+  templateUrl: './news-card.component.html',
+  styleUrls: ['./news-card.component.scss']
+})
+export class NewsCardComponent implements OnInit {
+  private detail: string;
+  private title: string;
+  private mediaPath: string;
+  @Input() private newsItem: any;
+
+  constructor(
+    private newsService: NewsService,
+  ) { }
+
+  ngOnInit() {
+    this.title = this.newsItem.title.rendered;
+    this.newsService.getNewsMediaThumbnailPath(this.newsItem.featured_media)
```

Comment:
Not one for this PR, but something to consider here is how much intelligence we add to components. Kind of ties into the link I sent over about atomic design, but also if we have dumb components that represent the design language which have minimal dependencies it would allow us to use [Storybook for managing UI](https://storybook.js.org).